### PR TITLE
cli: review command naming and help descriptions

### PR DESCRIPTION
**Pull request by lily-de**

### CODE REVIEW

**Comment by lilydelalande** [comment]
Screenshot 2024-11-26 at 4 07 04 PM
Screenshot 2024-11-26 at 4 07 33 PM
Screenshot 2024-11-26 at 4 07 19 PM

**Review by wendytang — APPROVED** [review]

**Comment by wendytang** [comment]
Merged the conflict, make sure it looks good before merging

File: crates/goose-cli/src/main.rs [hunk 1]
```diff
@@ -62,45 +62,136 @@ struct Cli {
...
+        )]
         profile: Option<String>,
-        #[arg(short, long, action = clap::ArgAction::SetTrue)]
+
+        /// Resume a previous session
+        #[arg(
+            short,
+            long,
+            help = "Resume a previous session (last used or specified by --session)",
+            long_help = "Continue from a previous chat session. If --session is provided, resumes that specific session. Otherwise resumes the last used session."
```
Comment (baxen) [inline crates/goose-cli/src/main.rs:114]: cc: @jsibbison-square to confirm the various combinations work as described here?

File: crates/goose-cli/src/main.rs [hunk 2]
```diff
@@ -62,45 +62,136 @@ struct Cli {
...
-        #[arg(short, long)]
+        /// Path to instruction file containing commands
+        #[arg(
+            short,
+            long,
+            required = true,
+            value_name = "FILE",
+            help = "Path to instruction file containing commands",
+            long_help = "Path to a file containing commands to execute. Each command should be on a new line. \
+                        The file will be executed in order from top to bottom."
```
Comment (baxen) [inline crates/goose-cli/src/main.rs]:
```suggestion
```

**Review by baxen — APPROVED** [review]

**Pull request merged**